ResourceStatusSystem/DB/ResourceManagementDB.py

```diff
@@ -501,6 +573,8 @@ def delete( self, table, params ):
           deleteQuery = deleteQuery.filter(column_a == columnValue)
         elif isinstance(columnValue, datetime.datetime):
           deleteQuery = deleteQuery.filter(column_a == columnValue)
+        elif isinstance(columnValue, bool):
+          deleteQuery = deleteQuery.filter(column_a == columnValue)
```

Comment:
I think this can be simplified:

```python
if isinstance(columnValue, (list, tuple)):
             select = select.filter(column_a.in_(list(columnValue)))
elif isinstance(columnValue, (basestring, datetime.datetime,bool)):
  select = select.filter( column_a == columnValue )
else:
 ...
```